cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/cluster/KafkaClusterOperations.java

```diff
@@ -102,7 +122,30 @@ public void create(String namespace, String name, Handler<AsyncResult<Void>> han
 
             result.add(statefulSetOperations.create(kafka.generateStatefulSet(isOpenShift)));
 
-            return CompositeFuture.join(result);
+            CompositeFuture
+                .join(result)
+                .compose(res -> statefulSetOperations.waitUntilReady(namespace, kafka.getName(), 1_000, 60_000))
```

Comment:
Something of an aside: I realise now that `waitUntilReady()` is the wrong name for that method. It doesn't block the caller, it returns a `Future` immediately.

Reply:
I'm fine with that but take it for another PR.